R/functions-Spectrum.R

```diff
@@ -979,10 +981,13 @@ meanMzInts <- function(x, ..., intensityFun = base::mean, weighted = FALSE,
         mz_groups <- .group_mz_values(sqrt(mzs), mzd = mzd)
     else
         mz_groups <- .group_mz_values(mzs, mzd = mzd, ppm = ppm)
-    if (length(unique(mz_groups)) < length(x[[main]]@mz))
-        warning("Got less m/z groups than m/z values in the original spectrum.",
-                " Most likely the data is not profile-mode LCMS data or ",
-                "'mzd' is too large.")
+    ## Disable the warning below, as this does not apply to all use cases: for
```

Comment:
Why not remove the whole commented block?

Reply:
Done